### PR TITLE
Add BNB support

### PR DESCRIPTION
**Pull request by anhang**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 cached-property==1.4.3
 xlrd==1.1.0
+pandas=*
```
Comment (omarish) [inline requirements.txt]: I think we should pin to a specific version of both pandas and requests.

File: data/exchange_rates.csv [hunk 2]
```diff
@@ -13778,3 +13778,187 @@ date,currency,rate
 2018-12-25,BTC,0.000242290346416
 2018-12-24,ETH,0.00782263479378
 2018-12-24,BTC,0.000251121111893
+
```
Comment (omarish) [inline data/exchange_rates.csv]: Are these newlines needed?

**Comment by anhang** [comment]
@omarish addressed your comments

**Comment by omarish** [comment]
Awesome, thanks @anhang! I don't have write permissions to this repo, but @justinmart should be able to take a look some time next week. Thanks again for the contribution.

**Pull request closed without merging**